docs/reference/relnotes/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Knative Release Notes"
+linkTitle: "Release notes"
+weight: 10
+type: "docs"
+---
```

Comment:
```suggestion
showlandingtoc: "false"
---
```

This hides the auto generated TOC list. Since there are not child rel notes topics, we can turn this off and hide it.

Reply:
this is still unresolved:

![image](https://user-images.githubusercontent.com/11762685/93245261-e4180080-f73f-11ea-8696-e71501d37371.png)

Reply:
just noticed that this is still waiting for action or feedback. do we want to keep this auto generated title because there are plans to add release notes in this section? otherwise we should add the frontmatter and get rid of it.

Reply:
This was reverted again? Do we want an auto gen TOC in the file? it has no children today

Reply:
TOC title is published here: https://knative.dev/development/reference/relnotes/